flask_app/resources/dish.py

```diff
@@ -1,6 +1,10 @@
 from flask_restful import Resource, reqparse, abort
 from models import db, Dishes
+import logging
 
+# logging configuration
+
+logging.basicConfig(filename='main.log', level=logging.DEBUG, format=('%(asctime)s %(levelname)s : %(message)s'))
```

Comment:
Shouldn't we create a place for `basicConfig` somewhere so we don't define it in each file that we want to use logger in ?